index.md

```diff
@@ -188,6 +197,9 @@ My own version of the [reference sheet](./CS245/ref.pdf){:.l#CS245--Reference-Sh
 from Ananya are available.
 These should be up-to-date with the latest course redesign.
 
+Detailed [lecture notes](https://grave-space-21c.notion.site/CS-246-7a97ec0fce9245798ca922f3404240fa?pvs=4) by Benny, Tommy and Yifei.
+If you want to contribute, feel free to contact us for edit permission!
```

Comment:
Maybe add your contact info to the bottom under the contact header instead of writing this here ("For resources by Benny, contact at . . .").